buildtest/cli/__init__.py

```diff
@@ -422,15 +422,15 @@ def history_menu(subparsers, parent_parser):
     """
 
     history_subcmd = subparsers.add_parser(
-        "history", aliases=["hy"], help="Query build history"
+        "history", aliases=["hy"], help="Query build history", parents=[parent_parser]
     )
 
     history_subparser = history_subcmd.add_subparsers(
         metavar="", description="Query build history file", dest="history"
     )
 
     list_parser = history_subparser.add_parser(
-        "list", help="List a summary of all builds", parents=[parent_parser]
+        "list", help="List a summary of all builds"
```

Comment:
so over here do `parents=[parent_parser]`. And might as well change the variable name `parent_parser` that is defined at top to something like `pager_option` or something. We may have a few global options that need to be shared between subcommands.

Reply:
@shahzebsiddiqui Done. Note that I have still kept the `--pager` option for `buildtest history query <id> --pager` and removed it for the `--output` option.